kloppy/infra/serializers/event/wyscout/serializer.py

```diff
@@ -0,0 +1,370 @@
+import json
+import logging
+from typing import Dict, List, Tuple
+
+from kloppy.domain import (
+    BallOutEvent,
+    BodyPart,
+    BodyPartQualifier,
+    CardEvent,
+    CardType,
+    EventDataset,
+    FoulCommittedEvent,
+    GoalkeeperAction,
+    GoalkeeperActionQualifier,
+    Ground,
+    Metadata,
+    Orientation,
+    PassEvent,
+    PassQualifier,
+    PassResult,
+    PassType,
+    Period,
+    PitchDimensions,
+    Player,
+    Point,
+    Provider,
+    Qualifier,
+    RecoveryEvent,
+    SetPieceQualifier,
+    SetPieceType,
+    ShotEvent,
+    ShotResult,
+    TakeOnEvent,
+    TakeOnResult,
+    Team,
+)
+from kloppy.infra.serializers.event import EventDataSerializer
+from kloppy.utils import Readable, performance_logging
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_team(raw_events: List[Dict], wyId: str, ground: Ground) -> Team:
+    team = Team(
+        team_id=wyId,
+        name=raw_events["teams"][wyId]["officialName"],
+        ground=ground,
+    )
+    team.players = [
+        Player(
+            player_id=str(p["playerId"]),
+            team=team,
+            jersey_no=None,
+            first_name=p["player"]["firstName"],
+            last_name=p["player"]["lastName"],
+        )
+        for p in raw_events["players"][wyId]
+    ]
+    return team
+
+
+def _has_tag(raw_event, tag_id) -> bool:
+    for tag in raw_event["tags"]:
+        if tag["id"] == tag_id:
+            return True
+    return False
+
+
+def _parse_shot(raw_event: Dict, next_event: Dict) -> Dict:
+    result = None
+    qualifiers = []
+    if _has_tag(raw_event, 101):
+        result = ShotResult.GOAL
+    elif _has_tag(raw_event, 2101):
+        result = ShotResult.BLOCKED
+    elif any((_has_tag(raw_event, t) for t in range(1217, 1224))):
```

Comment:
Please use meaningful variable name instead of ‘t’

Reply:
Done